datumaro/components/media.py

```diff
@@ -19,7 +20,22 @@
 BboxIntCoords = Tuple[int, int, int, int]  # (x, y, w, h)
 
 
+class MediaType(IntEnum):
+    NO_MEDIA = 0
+    UNKNOWN = 1
+    IMAGE = 2
+    BYTE_IMAGE = 3
+    VIDEO_FRAME = 4
+    VIDEO = 5
+    POINT_CLOUD = 6
+    MULTIFRAME_IMAGE = 7
+    ROI_IMAGE = 8
+    MOSAIC_IMAGE = 9
+
+
 class MediaElement:
+    MEDIA_TYPE = MediaType.UNKNOWN
```

Comment:
I am not sure why we need MEDIA_TYPE for each `MediaElement`. Could you explain this?

Reply:
This is because `MediaMapper` requires it. Each media should be assigned a corresponding integer value to infer its type when parsing from binary data.

Reply:
Aha okay, thanks.